### DISCUSSION TITLE
hw3-Induction Question

### DISCUSSION THREAD

**Discussion by Martin7-1** [courses-at-nju-by-hfwei/discrete-math-problem-sets#9] [Q&A] [answered]
老师我又来了...
对 **hw3-Induction**的第一题(也就是没有用上数学归纳法的那一道题)，这段证明我感觉还不是很明白...有几个小问题
1. 第一段证明是在证明引理，这里是将相识关系看做图的边来证明对吧？用到了上课提到的图论中常用的证明方法，是这样吗？
2. 对于第二段证明，是否有在这个大小为 $n+1$ 的 “好” 小组 $M$ 中，对任意在 $M$ 中的一个人，这个人都认识剩下的其他 $n$ 个人呢？这样不就没有办法找出存在的 $q$ 使他认识剩下的 $n$ 个人了？

![image](https://user-images.githubusercontent.com/77151639/122672342-be071b80-d1fd-11eb-83a1-ba782d7c0446.png)

**Comment by Martin7-1** [comment]
github 的这个Markdown居然不支持 Latex 的语法...

**Comment by hengxin** [comment, marked as answer]
@Martin7-1 (1) 可以认为是在用图论建模。但是这里比较简单，也没有用到图论的专门知识，可以不借助图论直接理解。(2) 已经证明了存在大小为 n+1 的好小组。一共 2n+1 个人。所以还剩下 n 个人。**根据题目条件**，存在不属于“剩下的这n个人”中的某个人与这n个人都认识。既然这个人不在剩下的这 n 个人中，那就是在大小为 n+1 的好小组里。所以这个人就符合条件。

**Reply by Martin7-1** [reply]
明白了...看来理解题目很重要。谢谢老师
